configuration/world.json

```diff
@@ -1,42 +1,67 @@
 {
-  "MasterCommunication": {
-	    "Password": "NosCorePassword",
+	"MasterCommunication": {
+		"Password": "NosCorePassword",
 		"Host": "http://localhost",
 		"Port": 5000,
 		"Routes": {
-			"ConnectedAccount" : "api/connectedAccount",
-			"Channel" : "api/channel",
-			"Session" : "api/session",
-			"Relation" : "api/relation",
-			"Packet" : "api/packet",
+			"ConnectedAccount": "api/connectedAccount",
+			"Channel": "api/channel",
+			"Session": "api/session",
+			"Relation": "api/relation",
+			"Packet": "api/packet",
 			"Stat": "api/stat",
-			"Friend" : "api/friend",
-			"Blacklist" : "api/blacklist"
+			"Friend": "api/friend",
+			"Blacklist": "api/blacklist"
 		}
-  },
-  "ServerName": "S1-NosCore",
-  "Host": "127.0.0.1",
-  "Port": 1337,
-  "ConnectedAccountLimit": 100,
-  "ServerGroup": 0,
-  "WorldInformation": true,
-  "WebApi": {
+	},
+	"ServerName": "S1-NosCore",
+	"Host": "127.0.0.1",
+	"Port": 1337,
+	"ConnectedAccountLimit": 100,
+	"ServerGroup": 0,
+	"WorldInformation": true,
+	"WebApi": {
 		"Host": "http://localhost",
 		"Port": 5001,
 		"Password": "NosCorePassword"
-  },
-  "Language" : "en",
-  "Database": {
-	  "Host" : "localhost",
-	  "Port" : 5432,
-	  "Database" : "postgres",
-	  "Username" : "postgres",
-	  "Password" : "password",
 	},
-	"BackpackSize":48,
-	"MaxItemAmount":999,
-	"MaxGoldAmount":1000000000,
-	"MaxBankGoldAmount":100000000000,
-	"MaxSpPoints":10000,
-	"MaxAdditionalSpPoints":1000000
+	"Language": "en",
+	"Database": {
+		"Host": "localhost",
+		"Port": 5432,
+		"Database": "postgres",
+		"Username": "postgres",
+		"Password": "password"
+	},
+	"BackpackSize": 48,
+	"MaxItemAmount": 999,
+	"MaxGoldAmount": 1000000000,
+	"MaxBankGoldAmount": 100000000000,
+	"MaxSpPoints": 10000,
+	"MaxAdditionalSpPoints": 1000000,
+	"SumSandVNum": 1027,
```

Comment:
i don't want this in world conf and certainly not in multiple object when it can be set as a big one.
in the idea just use an helper 
so we can move it when doing https://github.com/NosCoreIO/NosCore/issues/726